<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>EBCDIC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Binary Header</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_Model">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuModel">
     <property name="title">
      <string>SEGY</string>
     </property>
     <addaction name="actionExplore_seq"/>
     <addaction name="actionExplore_SEGY_by_groups"/>
     <addaction name="actionCreate_model"/>
     <addaction name="actionConvert_SEGD_to_SEGY"/>
     <addaction name="actionCreate_geometry_from_model"/>
    </widget>
    <widget class="QMenu" name="menuSU">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>SU</string>
     </property>
     <addaction name="actionExplore_SU"/>
    </widget>
    <addaction name="menuModel"/>
    <addaction name="menuSU"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuOpen_Model"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_model">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create model</string>
   </property>
  </action>
  <action name="actionExplore_seq">
   <property name="text">
    <string>Explore SEGY sequentially</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConvert_SEGD_to_SEGY">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Convert SEGD to SEGY</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCreate_geometry_from_model">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create geometry from model</string>
   </property>
  </action>
  <action name="actionExplore_SEGY_by_groups">
   <property name="text">
    <string>Explore SEGY by groups</string>
   </property>
  </action>
  <action name="actionExplore_SU">
   <property name="text">
    <string>Explore SU</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
